metrics-core/pom.xml

```diff
@@ -51,6 +51,30 @@
       <version>${trove4j.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>4.0.23.Final</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.cassandra</groupId>
+      <artifactId>cassandra-clientutil</artifactId>
+      <version>2.1.1</version>
+    </dependency>
+
+    <dependency>
```

Comment:
It is a mistake on my side to not clean the pom properly. Earlier in the development cycle, I had the code in the metrics-core to get things going. I will fix it.